app/lib/shared/configuration.dart

```diff
@@ -100,6 +100,15 @@ class Configuration {
   /// **Optional**, if omitted email sending is disabled.
   final String gmailRelayImpersonatedGSuiteUser;
 
+  /// The email of the service account which has access rights to sign upload
+  /// requests.
+  ///
+  /// Authorization requires the following IAM permission on the specified resource name:
+  /// - iam.serviceAccounts.signBlob
```

Comment:
Might be worth mentioning that this is on the package bucket..

Reply:
and importantly that the service account running this service MUST be able to impersonate this account.